examples/platforms/efr32mg21/flash.c

```diff
@@ -32,68 +32,92 @@
  *   storage for the EFR32 platform using the Silabs Nvm3 interface.
  */
 
+#include "openthread-core-efr32-config.h"
 #include <openthread/config.h>
-#include <openthread/platform/settings.h>
 
-#if !OPENTHREAD_CONFIG_PLATFORM_FLASH_API_ENABLE
+#if OPENTHREAD_CONFIG_PLATFORM_FLASH_API_ENABLE // Defaults to OT NV system
 
-#include "nvm3.h"
-#include "nvm3_hal_flash.h"
+#include "em_msc.h"
 #include <string.h>
-#include "common/code_utils.hpp"
+#include <openthread/instance.h>
+
+#define FLASH_PAGE_NUM 4
```

Comment:
I would recommend using #define FLASH_PAGE_NUM 2 here since this is the number of flash pages used in the previous versions of the built in nv manager.

Reply:
Fixed it. Thank you.